packages/components/psammead-brand/src/index.jsx

```diff
@@ -122,10 +122,11 @@ const BrandSvg = styled.svg`
   /* stylelint-enable */
 `;
 
-const LocalisedBrandName = ({ product, serviceLocalisedName }) =>
+const LocalisedBrandName = ({ linkId, product, serviceLocalisedName }) =>
   serviceLocalisedName ? (
+    // id={`BrandLink-${linkId}` is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
```

Comment:
I had a look at the issue. Can I ask why this is a temporary fix? Is it only temporary until Google/TalkBack provide a fix or is it a temporary fix for us over a fixed period? Thanks

Reply:
until Google/Talkback fixes it. But we think they are going to take a very long time. I will specify this in the issue link.

Reply:
Well it could take a long time for Google to fix, this bug has been present for some time, though we hope they might fix it soon, it's out of our control.